Gemfile

```diff
@@ -62,7 +62,7 @@ end
 # Do not place the capistrano-related gems in the default or Rails.env bundle group
 # Otherwise the config/application.rb's Bundle.require command will try to load them
 # leading to failure because these gem's rake task files use capistrano DSL.
-group :deployment do
+group :development do
```

Comment:
isn't this really deployment?  capistrano gems?

Reply:
No, we run capistrano in the rails development (default) environment. So "development" is appropriate.

Reply:
but if you want to not include the cap gems in the deployed version, deployment is correct (bundle without --deployment).   And cap still installs locally on your laptop (where you use it) because then you keep the deployment group by default.  This is the way I've seen it done (and have been using on other projects)

Reply:
We use the `:deployment` group for `hydra_etd`... but I'm not sure why we need it. `:development` should be fine, methinks? Deployed apps tend to run in the `:production` env (or, at least, not in the `:development` env, right?), so they wouldn't pick up these dependencies either before or after this change.